### ISSUE TITLE
New version of GrooveShark

### ISSUE THREAD

**Issue by fernando-takeshi** [xolarity/WinGrooves#34]
Are there any plans to update to the latest version of GrooveShark?

**Comment by gueyeb** [comment]
+1

**Comment by stvndall** [comment]
I pulled the code to see if I could sort out the problem, and it seems to work on the latest code, Please try re download. see if that fixes it. Grooveshark looks for the buttons by ID, which all still seem to be correct. it could be Grooveshark had temporarily changed the names of their buttons, or newest build has some fixes from previous that help out with the change over to the new Grooveshark :)

**Comment by rdaunce** [comment]
I was running version 1.0.8.0 and hotkeys weren't working.  I downloaded the latest and it upgraded to version 1.0.9.0 and hotkeys functioned properly.  Make sure you are running the latest version and I'm sure things will work properly.

**Comment by fernando-takeshi** [comment]
Yeah...updating the version helped, although I think I observed some odd behavior: clicking the new interface for shuffle / loop seemed to desynchronized the checkboxes' status in the context menu of the system tray icon. Minor nuisance, but still...

Also, it seems that the 'Are you still there -> click here to resume' made a comeback in the new version x(
